src/Components/Artwork/Metadata.tsx

```diff
@@ -65,7 +63,7 @@ export const Metadata: React.FC<MetadataProps> = ({
 }
 
 const DisabledLink = styled(RouterLink)`
-  cursor: not-allowed;
+  cursor: default;
 `
```

Comment:
This method of disabling the link is _not good_. We shouldn't have any empty anchor tag hanging out like this — it's still keyboard focusable.

Reply:
You should likely completely swap out the tag here.

Reply:
This is a good example of "fixing" the symptom rather than the root cause.